homeassistant/components/fronius/sensor.py

```diff
@@ -68,7 +75,7 @@ def _device_id_validator(config):
     session = async_get_clientsession(hass)
     fronius = Fronius(session, config[CONF_RESOURCE])
 
-    sensors = []
+    scan_interval = config[CONF_SCAN_INTERVAL]
```

Comment:
We can use dict.get here to set a default interval.

Reply:
Okay. It just confused me that `get` would return None if no scan interval is given. I somehow tried to get a system wide set default scan interval at first but that doesn't seem to exist.
Thanks!